card/mod/history/lib/card/act/act_renderer/relative_act_renderer.rb

```diff
@@ -46,6 +46,11 @@ def current_act?
         def actions
           @actions ||= @act.actions_affecting(@card)
         end
+
+        def revert_link
+          revert_link_to "this", revert_actions: actions.map(&:id),
```

Comment:
because we call `actions` here (as opposed to `act.actions` in the format), this is now limited only to the relative actions desired.